src/librbd/cache/pwl/Builder.h

```diff
@@ -0,0 +1,86 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_LIBRBD_CACHE_PWL_BUILDER_H
+#define CEPH_LIBRBD_CACHE_PWL_BUILDER_H
+
+#include <iostream>
+#include "SSDLogEntry.h"
+#include "RWLLogEntry.h"
+#include "librbd/cache/ImageWriteback.h"
+
+#define dout_subsys ceph_subsys_rbd_pwl
+#undef dout_prefix
+#define dout_prefix *_dout << "librbd::cache::pwl::Builder: " << this << " " \
+                           <<  __func__ << ": "
+
+namespace librbd {
+
+namespace cache {
+
+namespace pwl {
+
+class Builder {
+public:
+  virtual ~Builder() {}
+  virtual std::shared_ptr<WriteLogEntry> createWriteLogEntry(
```

Comment:
Nit: please use snake-case instead of camel-case to keep things consistent.

Reply:
Done